myapp/templates/myapp/setting.html

```diff
@@ -1 +1,24 @@
-S
\ No newline at end of file
+{% extends 'myapp/base.html' %}
+{% load static %}
+{% block extra_style %}
+<link rel="stylesheet" href="{% static 'myapp/css/setting.css' %}">
+{% endblock %}
+<body>
+    {% block header %}
+    <header class="header">
+        設定
+    </header>
+    {% endblock %}
+    {% block content %}
+    <button class="login-btn"><a href="{% url 'namechange' %}">ユーザ名変更</a></button>
+    <button class="login-btn"><a href="{% url 'mailchange' %}">メールアドレス変更</a></button>
+    <button class="login-btn"><a href="{% url 'imagechange' %}">アイコン変更</a></button>
+    <button class="login-btn"><a href="{% url 'passwordchange' %}">パスワード変更</a></button>
+    <form method="post" action="{% url 'logout' %}">
+        {%csrf_token %}
+        <div class="logout-btn">
+            <button  type="submit">ログアウト</button>    
+        </div>
+    </form>
```

Comment:
以下のようにflexを利用してスタイリングすると中央揃えを行うことが可能です。
flexによるスタイリングはかなり実務でも使用するので調べて使い方を覚えておくと便利です。

```suggestion
<div style="display: flex; flex-direction: column; gap: 24px; align-items: center; padding: 8px;">
    <button class="login-btn"><a href="{% url 'namechange' %}">ユーザ名変更</a></button>
    <button class="login-btn"><a href="{% url 'mailchange' %}">メールアドレス変更</a></button>
    <button class="login-btn"><a href="{% url 'imagechange' %}">アイコン変更</a></button>
    <button class="login-btn"><a href="{% url 'passwordchange' %}">パスワード変更</a></button>
    <form method="post" action="{% url 'logout' %}">
        {%csrf_token %}
        <div class="logout-btn">
            <button  type="submit">ログアウト</button>    
        </div>
    </form>
</div>
```